org.onlyoffice.desktopeditors.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id": "org.onlyoffice.desktopeditors",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "desktopeditors",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-desktop",
+        "--filesystem=xdg-videos",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--device=all"
+    ],
+    "modules": [
+        {
+            "name": "desktopeditors",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 desktopeditors /app/bin/desktopeditors",
+                "cp /usr/bin/ar /app/bin",
+                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib",
+                "install -Dm755 org.onlyoffice.desktopeditors.metainfo.xml /app/share/metainfo/org.onlyoffice.desktopeditors.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "ar x onlyoffice-desktopeditors_amd64.deb",
+                        "tar xf data.tar.xz",
+                        "mkdir -p export/share/applications",
+                        "sed 's|Icon=.*|Icon=org.onlyoffice.desktopeditors|; s|Exec=.*|Exec=desktopeditors|;' usr/share/applications/desktopeditors.desktop > export/share/applications/org.onlyoffice.desktopeditors.desktop",
+                        "echo 'StartupWMClass=onlyoffice' >> export/share/applications/org.onlyoffice.desktopeditors.desktop",
+                        "for icon_size in 16 24 32 48 64 128 256; do
+                            install -Dm644 opt/onlyoffice/desktopeditors/asc-de-${icon_size}.png export/share/icons/hicolor/${icon_size}x${icon_size}/apps/org.onlyoffice.desktopeditors.png
+                        done",
+                        "rm -r onlyoffice-desktopeditors_amd64.deb control.tar.gz data.tar.xz debian-binary usr"
+                    ]
+                },
+                {
+                    "type": "extra-data",
```

Comment:
https://github.com/flathub/flathub/pull/1247/files There's another PR doing exactly that, but yours is preferred as you're from upstream team.

Reply:
@barthalion, you are right absolutely, and we are already working on it, but I'm afraid we stack on building 3rd party components (like a v8) for a long time.

Reply:
Yeah, that's fine, I don't mean building from source. You could just re-use binary artifact like Discord does: https://github.com/flathub/com.discordapp.Discord/blob/master/com.discordapp.Discord.json#L29

We can merge it as is if it will get changed away from extra-data eventually (unless you don't want to implicitly grant us redistribution rights etc, that's also fine).

Reply:
@barthalion, I have to change source type form `extra-data` to `archive` and use tar.gz package instead of deb, isn't it?

Reply:
Yeah, that would be great. I merged the app in the meantime.